src/EditPopup.js

```diff
@@ -0,0 +1,32 @@
+import React, {useCallback, useState} from 'react';
+import './Popup.css';
+
+
+const EditPopup = ({show, closeEdit, onSubmit}) => {
+    const [value, setValue] = useState("");
+
+    const handleChange = useCallback(event => setValue(event.target.value), []);
+    const handleSubmit = useCallback((e) => {
+        e.preventDefault();
+        onSubmit(value);
+        setValue("");
+        closeEdit();
+    }, [value, onSubmit, closeEdit]);
+
+    return(
+        <div className={"editWrapper" + (show ? '' : ' hidden')}>
+            <div className={'edit'} id="editPopup">
+                <h1 className="editHeader">Would you like to change your Idea?</h1>
+                <form onSubmit={handleSubmit}>
+                    <label>New Idea:</label><br/>
+                    <input type="text" value={value} onChange={handleChange} /><br/>
+                    <input className="submitButton" type="submit" value="Yes Please!"/>
+                </form>
+                <button onClick={closeEdit}>No Thanks</button>
+                
+            </div>
+        </div>
+    );
+}
+
+export default EditPopup;
```

Comment:
It's good practice to end your file with a newline. My editor automatically does it for me. The newline character is technically at the end of the last line so if you don't have one at the end, and ever decide to add more lines after the end, git will show it as a line change.